static/js/utils.js

```diff
@@ -1,14 +1,16 @@
+/* exported makeEls */
 
-function makeEl(s, text, attr) {
-    var a = s.split('.');
-    var tag = a.shift();
-    var el = document.createElement(tag);
-    el.className = a.join(' ');
-    if (text) el.textContent = text;
-    if (attr) {
-        for (var k in attr) {
-            el.setAttribute(k, attr[k]);
-        }
+var makeEls = function (s, text, attr) {
```

Comment:
I think this has to be `var`, since we are specifically trying to expose it to other files. Changing this to `let`/`const` would make it only visible in this file, which wouldn't work since we aren't using ES6 modules yet. (should we?)